CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Changes by Version
 ### Backend Changes
 
 #### Breaking Changes
+* Add arm64 support for binaries and docker images. Rename the released binary name format to <Name>-<OS>-<ARCH>, tarball name format to jaeger-<Version>-<OS>-<ARCH> ([#2176](https://github.com/jaegertracing/jaeger/pull/2176), [@MrXinWang](https://github.com/MrXinWang))
```

Comment:
The released binary names or tarball names were not changed. Or am I missing something? 

The build names via `make build-` is an internal detail, hence we don't have to list this as a breaking change.

Reply:
`Add arm64 support for binaries and make architecture configurable in docker images.`

Reply:
The tarball names have not changed, but binary names inside them have changed to include arch.

Reply:
The binary names seem the same as before:

```
Packaging into jaeger-1.18.0-linux-amd64.tar.gz:
jaeger-1.18.0-linux-amd64/
jaeger-1.18.0-linux-amd64/jaeger-agent
jaeger-1.18.0-linux-amd64/jaeger-all-in-one
jaeger-1.18.0-linux-amd64/example-hotrod
jaeger-1.18.0-linux-amd64/jaeger-query
jaeger-1.18.0-linux-amd64/jaeger-ingester
jaeger-1.18.0-linux-amd64/jaeger-collector

Packaging into jaeger-1.18.0-darwin-amd64.tar.gz:
jaeger-1.18.0-darwin-amd64/
jaeger-1.18.0-darwin-amd64/jaeger-agent
jaeger-1.18.0-darwin-amd64/jaeger-all-in-one
jaeger-1.18.0-darwin-amd64/example-hotrod
jaeger-1.18.0-darwin-amd64/jaeger-query
jaeger-1.18.0-darwin-amd64/jaeger-ingester
jaeger-1.18.0-darwin-amd64/jaeger-collector

Packaging into jaeger-1.18.0-windows-amd64.tar.gz:
jaeger-1.18.0-windows-amd64/
jaeger-1.18.0-windows-amd64/jaeger-agent.exe
jaeger-1.18.0-windows-amd64/jaeger-ingester.exe
jaeger-1.18.0-windows-amd64/jaeger-all-in-one.exe
jaeger-1.18.0-windows-amd64/jaeger-query.exe
jaeger-1.18.0-windows-amd64/example-hotrod.exe
jaeger-1.18.0-windows-amd64/jaeger-collector.exe
```

Reply:
Hi @pavolloffay ! I think now the x86_64 binaries are added an `amd64` after `linux`, before they are just <name>-linux. Arm64 and s390x ones are the same. This can be derived from the Makefile changes, for example: [here](https://github.com/jaegertracing/jaeger/pull/2176/files#diff-b67911656ef5d18c4ae36cb6741b7965L232).

Reply:
maybe we strip all suffixes before creating tarballs?

Reply:
@yurishkuro I checked the `scripts/travis/package-deploy.sh` and it turns out that @pavolloffay is right, there is a staging step where we rename suffixes of all binaries...So I think I can remove the breaking change in CHANGELOG.md